testutils/handler.go

```diff
@@ -0,0 +1,31 @@
+package testutils
+
+import (
+	"sync"
+	"time"
+
+	"github.com/segmentio/stats"
+)
+
+var _ stats.Handler = (*Handler)(nil)
+
+type Handler struct {
```

Comment:
There are couple of sub-packages that have redeclared a type this this over and over, do you mind following up with the PR to port those to use this type?